document-store-service/src/main/java/dk/dbc/search/solrdocstore/DocStoreApplication.java

```diff
@@ -13,14 +13,15 @@
      * This class defines the other classes that make up this JAX-RS application
      * by having the getClasses method return a specific set of resources.
      */
-@ApplicationPath("/")
+@ApplicationPath("")
 public class DocStoreApplication extends Application {
         private static final Logger log = LoggerFactory.getLogger(DocStoreApplication.class);
 
         @Override
         public Set<Class<?>> getClasses() {
             final Set<Class<?>> classes = new HashSet<>();
```

Comment:
Lille ting - men vi har konstateret at getClasses() bliver kaldt relativt ofte, nogle gange endda mere end een gang for den samme stateless bean instans, så for at undgå det lille overhead bruger vi nu typisk i dataIO

public static final Set<Class<?>> classes = new HashSet<>();
static {
    classes.add(XYZ.class);
}

@Override
 public Set<Class<?>> getClasses() {
     return classes;
}